simpleflow/canvas.py

```diff
@@ -3,6 +3,7 @@
 from .base import Submittable
 from .signal import WaitForSignal
 from .task import ActivityTask, SignalTask
+from .step import Step
 
 
 def propagate_attribute(obj, attr, val):
```

Comment:
"Can steps raises_on_failure": er, let's say I meant: can we/do we wish to propagate inside steps? 🙂